curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/step-036.md

```diff
@@ -7,7 +7,7 @@ dashedName: step-36
 
 # --description--
 
-The `input` element allows you several ways to collect data from a web form. Like anchor (`a`) elements, `input` elements are <dfn>self-closing</dfn> and do not need closing tags.
+The `input` element allows you several ways to collect data from a web form. Like images (`img`) elements, `input` elements are <dfn>self-closing</dfn> and do not need closing tags.
```

Comment:
In the second sentence, we should change images (`img`) elements to just say `img` elements. 

Proposed change: Like `img`elements,